Doc/library/functools.rst

```diff
@@ -430,6 +430,17 @@ The :mod:`functools` module defines the following functions:
      ...     for i, elem in enumerate(arg):
      ...         print(i, elem)
 
+   .. note::
+      The :mod:`typing` module provides a number of aliases to
+      builtin classes, as well as classes found elsewhere in the standard
+      library. All have been deprecated since Python 3.9 (see :pep:`585`).
```

Comment:
It would be good to clarify that this usage doesn't work since 3.9 because usually deprecation means something is planned to stop working in the future, not at the time it was deprecated.